README.md

```diff
@@ -32,8 +32,9 @@ The app launches a window displaying a map.
 ### Command Line
 
 1. `cd` into the project's root directory.
-2. Run `./gradlew clean build` on Linux/Mac or `gradlew.bat clean build` on Windows once to make sure the dependencies are fetched.
-3. Run `./gradlew run` on Linux/Mac or `gradlew.bat run` on Windows to run the app.
+2. Run `gradle wrapper` to build a fresh version of `./gradle/wrapper/gradle-wrapper.jar`.
```

Comment:
Having Gradle installed rather defeats the purpose of the Gradle wrapper. The jar should come with this app.

Reply:
then the project must add `gradle-wrapper.jar` because it was not there and this project fails to build out of the box. I assumed it was left out so that the developer was to build it themselves.

Reply:
I just talked to @tschie  , it should have that jar file included. He'll check it out tomorrow.

Reply:
@jf990 Sorry, didn't realize the jar was missing, it's supposed to be in that Gradle folder! I'll fix it now.

Reply:
The gitignore file that GitHub generates for Java projects ignores all jars. That's why it was missing. I've made an exception for it. Should be fixed now!

Reply:
@jf990 we merged in a .gitignore, the jar file, and added execution permissions to the wrapper file if you want to give it a test from master.

Reply:
i tested it on a clean system and it works like a charm. thanks!